python/ql/src/experimental/Security/CWE-611/XXE.qlref

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Parsing untrusted XML files with a weakly configured XML parser may lead to an XML External Entity (XXE) attack.
+This type of attack uses external entity references to access arbitrary files on a system, carry out denial of
+service, or server side request forgery. Even when the result of parsing is not returned to the user, out-of-band
+data retrieval techniques may allow attackers to steal sensitive data. Denial of services can also be carried out
+in this situation.
+</p>
+<p>
+Refer to the following links to check the details regarding how and which libraries are vulnerable:
```

Comment:
Thanks for making the changes. I think it would still be useful to mention the sinks that this query is trying to catch. (i.e. To restore
```
This query currently identifies vulnerable XML parsing from the following parsers: 
<code>xml.etree.ElementTree.XMLParser</code>, <code>lxml.etree.XMLParser</code>, <code>lxml.etree.get_default_parser</code>,
<code>xml.sax.make_parser</code>.
```
from the original doc, which will make it more clear what this query is doing and the users can look up the parsers against the links more easily)
Thanks!

Reply:
Great 👍